engine/apps/integrations/urls.py

```diff
@@ -33,6 +34,7 @@
     path("alertmanager/<str:alert_channel_key>/", AlertManagerAPIView.as_view(), name="alertmanager"),
     path("amazon_sns/<str:alert_channel_key>/", AmazonSNS.as_view(), name="amazon_sns"),
     path("heartbeat/<str:alert_channel_key>/", HeartBeatAPIView.as_view(), name="heartbeat"),
+    path("alertmanager/v2/<str:alert_channel_key>/", AlertManagerV2View.as_view(), name="alertmanager_v2"),
```

Comment:
```suggestion
    path("alertmanager_testing/<str:alert_channel_key>/", AlertManagerV2View.as_view(), name="alertmanager_v2"),
```

may be call it alertmanager_testing - so nobody starts using it